cli/print.go

```diff
@@ -43,7 +43,7 @@ func infof(w io.Writer, format string, a ...interface{}) {
 // do not currently make use of the variadic `a` parameter but may in the
 // future. unparam will complain until it does.
 //
-//nolint:unparam
```

Comment:
sweet, if we don't need to disable this rule anymore let's remove the `NOTE` above as well.

Reply:
Agree! And, apologies for the `NOTE(name)` format there; will stop doing this unless the code really represents a controversial opinion.